static/css/event.less

```diff
@@ -0,0 +1,102 @@
+/* ==========================================================================
+   cfgov-refresh
+   Event post detail page
+   ========================================================================== */
+
+.event-status {
```

Comment:
I'm not sure if page-specific styles need topdoc docs or not. I think they were being left off for page-specific styles. E.g. https://github.com/cfpb/cfgov-refresh/blob/gh-pages/static/css/budget.less